src/swupd-build-opts.h

```diff
@@ -46,7 +46,73 @@
 #define OPT_BSDTAR "-BSDTAR"
 #endif
 
+#ifdef SWUPD_WITH_XATTRS
+#define OPT_XATTRS "+XATTRS"
+#else
+#define OPT_XATTRS "-XATTRS"
+#endif
+
+#ifdef SWUPD_TAR_SELINUX
+#define OPT_SELINUX "+TAR_SELINUX"
+#else
+#define OPT_SELINUX "-TAR_SELINUX"
+#endif
+
+#ifdef OS_IS_STATELESS
+#define OPT_STATELESS "+STATELESS"
+#else
+#define OPT_STATELESS "-STATELESS"
+#endif
+
+#ifdef CONTENTURL
+#define OPT_C_URL CONTENTURL
+#else
+#define OPT_C_URL "no default set"
+#endif
+
+#ifdef VERSIONURL
+#define OPT_V_URL VERSIONURL
+#else
+#define OPT_V_URL "no default set"
+#endif
+
+#ifdef FORMATID
+#define OPT_FORMAT FORMATID
+#else
+#define OPT_FORMAT "no default set"
+#endif
+
+#ifdef SYSTEMD_UNITDIR
+#define OPT_SYSTEMD SYSTEMD_UNITDIR
+#else
+#define OPT_SYSTEMD "no default set"
```

Comment:
Should this be included in BUILD_CONFIGURE? I don't see this used anywhere.

Reply:
You are right. Removed from this commit and I added a commit to remove it from configure.ac and Makefile.am since it looks like the option isn't used anywhere anymore.